packages/aws-cdk/lib/init-templates/app/dotnet/README.md

```diff
@@ -0,0 +1,20 @@
+
+# CDK HelloWorld
+
+Welcome to your CDK .NET project!
+
+You should explore the contents of this template. It demonstrates a CDK app with two instances of
+a stack (`HelloStack`) which also uses a user-defined construct (`HelloConstruct`).
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app. It uses a script called `app.sh`
```

Comment:
`app.sh` on Windows?

Reply:
For now I'm following the pattern from the Java init template. Other languages vs .NET is orthogonal to OS--we should provide cross-platform scripts in every template.

Reply:
Changed to invoke the `dotnet` CLI directly, rather than through a script.
✔️